### PR TITLE
Buff Blindsight, Nerf Skeletal Piglin

### PR DESCRIPTION
**Pull request by Dariensg**
-Skeletal Piglins have a much lower chance to spawn with a shovel (from 70% to 30%)
-Blindsights now have more of a range in delay between jumps
-Blindsights now have variable jump heights

### CODE REVIEW

File: src/main/java/com/nekomaster1000/infernalexp/entities/BlindsightEntity.java [hunk 1]
```diff
@@ -111,7 +113,17 @@ protected void jump() {
...
+
+        Vector3d vector3d = this.getMotion();
+        this.setMotion(vector3d.x, f, vector3d.z);
+        if (this.isSprinting()) {
+            float f1 = this.rotationYaw * ((float)Math.PI / 180F);
+            this.setMotion(this.getMotion().add((-MathHelper.sin(f1) * 0.2F), 0.0D, (MathHelper.cos(f1) * 0.2F)));
+        }
+
+        this.isAirBorne = true;
+        net.minecraftforge.common.ForgeHooks.onLivingJump(this);
```
Comment (SwanX1) [inline src/main/java/com/nekomaster1000/infernalexp/entities/BlindsightEntity.java]: what does this do, this doesn't have any comments

Comment (nekomaster1000) [reply src/main/java/com/nekomaster1000/infernalexp/entities/BlindsightEntity.java:126]: Shut up swan you’re not a dev

Comment (Hellion-Games) [reply src/main/java/com/nekomaster1000/infernalexp/entities/BlindsightEntity.java]: It's the code from super.jump(), but moved up to allow us to modify it easily. I added a random value to the calculation of f to get a range of jump heights for the Blindsight to give it a little bit more character.

**Comment by Dariensg** [comment]
-Commented out All Non-Release 2 Content

**Pull request merged**